### PR TITLE
AFM 1.17: Changes to the documentation

### PR DESCRIPTION
**Pull request by rddheath**
I still need to write up how to use private locations from the BAT CLI. I've include the location commands in the BAT command reference, but need to explain how to run tests in private locations. I'll add that by tomorrow morning and let you all know in afm-product-leaders.

### CODE REVIEW

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+When you add a schedule to a monitor, you can use the schedule to run any of the versions of the test for that monitor.
+
+== Procedure
+
+To add a schedule from the *Schedules* section of a monitor:
+
+. Select the monitor in which you want to add a schedule.
+. Click the *Schedules* tab.
+. Click *Add Schedule*.
+. In the Add Schedule dialog, specify this information:
```
Comment (runnabro) [inline modules/ROOT/pages/afm-add-schedule.adoc]: Should this "Add Schedule" be bold too?

File: modules/ROOT/pages/afm-add-schedule.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
++
+* The test version that you want the schedule to run
+* The location to run the test from
+* The frequency at which to run the test
+. Click *Add*.
```
Comment (runnabro) [inline modules/ROOT/pages/afm-add-schedule.adoc:29]: Once the form is visible, is this level of detail necessary? I'd rather see this space used to tell users how to fill out the form (reference to private locations, or a reference for how to write Cron).

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
  ** xref:afm-create-monitor.adoc[Create a Monitor]
+ ** xref:afm-create-private-location.adoc[Create a Private Location]
+ ** xref:afm-add-schedule.adoc[Add a Schedule]
+ ** xref:afm-modify-schedule.adoc[Modify a Schedule]
+ ** xref:afm-delete-schedule.adoc[Delete a Schedule]
+ ** xref:afm-view-schedules-private-location.adoc[View the Schedules That Use a Private Location]
+ ** xref:afm-delete-private-location.adoc[Delete a Private Location]
+ ** xref:afm-delete-test.adoc[Delete a Version of a Test]
+ ** xref:afm-run-test-now.adoc[Run a Test Immediately]
+ ** xref:afm-view-test-history.adoc[View a Test's History]
```
Comment (runnabro) [inline modules/ROOT/nav.adoc:14]: This is a single quote, but you should use an apostrophe `’`

File: modules/ROOT/nav.adoc [hunk 4]
```diff
@@ -1,7 +1,19 @@
 .xref:index.adoc[API Functional Monitoring]
 * xref:index.adoc[About API Functional Monitoring]
+* xref:afm-public-versus-private.adoc[Differences in the Monitoring of the Endpoints of Public and Private APIs]
```
Comment (runnabro) [inline modules/ROOT/nav.adoc]: This title is a bit wordy to me

File: modules/ROOT/pages/bat-command-reference.adoc [hunk 5]
```diff
@@ -52,118 +52,126 @@ Executes the desired folder. BAT searches for and executes the `bat.yaml` and `m
...
-Profile is:
-+
-*** The name of a profile for accessing an Anypoint Platform location and retaining the login information between BAT sessions. Using this option eliminates the need to relogin between sessions. Optional.
-*** The name of a profile for scheduling an endpoint that can be different from the BAT login profile you used. By default, BAT uses the login profile for scheduling. Optional.
-+
-** `--init-folder`
-+
-Stores the files, exchange.json and main.dwl, generated by creating the monitor in the current directory. Optional.
-** `--init`
+** `target ls list`
```
Comment (diegor) [inline modules/ROOT/pages/bat-command-reference.adoc]: the 'target' command has been renamed to location. Remove this line

File: modules/ROOT/pages/afm-add-schedule.adoc (hunk 2 above)
Comment (rddheath) [reply modules/ROOT/pages/afm-add-schedule.adoc:29]: @runnabro , why would you like the doc to explain how to write cron jobs? The UI doesn't require users to do write them. Also, what private locations are is described in a separate topic. I'll add a link to that topic.

File: modules/ROOT/nav.adoc (hunk 3 above)
Comment (rddheath) [reply modules/ROOT/nav.adoc:14]: @runnabro , that is an apostrophe in a monotype font.

**Review by dukesphere — APPROVED** [review]
